deploy/grafana-dashboard/runner_insights_row.py

```diff
@@ -0,0 +1,66 @@
+from grafanalib.core import RowPanel, GridPos, Histogram, TimeSeries
+from grafanalib.influxdb import InfluxDBTarget
+
+from util import read_query
+
+execution_duration_extra_json = {
+    "fieldConfig": {
+        "defaults": {
+            "unit": "ns"
+        }
+    }
+}
+execution_duration = Histogram(
+    title="Execution duration",
```

Comment:
Here, I am also wondering why we see a difference between the old and new dashboard, for example with regard to the scale of the y axis (only ~15s instead of ~30s)

![Bildschirm­foto 2022-10-27 um 00 03 06](https://user-images.githubusercontent.com/7300329/198149029-7ae82483-5521-44dd-a25c-2d8cba24cd8b.png)

Reply:
Like for the other panels, this is a result of the static number of `maxDataPoints`. With less displayed data points, all data points need to get more aggregated and so the mean of them is lower.

I have now changed it to a dynamic number again.